package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "dav-client",
+  "version": "0.0.1",
+  "description": "A DAV client to connect to DAV servers",
+  "main": "build/main.js",
+  "scripts": {
+    "build": "rollup -c",
+    "dev": "rollup -c -w"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^17.1.0",
+    "@rollup/plugin-node-resolve": "^11.2.0",
+    "@rollup/plugin-typescript": "^8.2.0",
+    "@tsconfig/recommended": "^1.0.1",
+    "@types/url-join": "^4.0.0",
+    "rollup": "^2.42.0",
```

Comment:
We'll use webpack as it's the standard here

Reply:
Unfortunately, Webpack doesn't support output-ing ES Modules (which is crucial for tree shaking and code splitting later on when we build our app): https://github.com/webpack/webpack/issues/2933. So we still need to use Rollup here.

Reply:
vite is also in the table too https://vitejs.dev/

Reply:
Vite is mostly for applications. It also uses Rollup to build under the hood.